packages/quantic/force-app/examples/main/lwc/exampleUseCase/exampleUseCase.js

```diff
@@ -0,0 +1,16 @@
+import {api, LightningElement} from 'lwc';
+
+export default class ExampleUseCase extends LightningElement {
+  @api engineId = '';
+  @api useCase = '';
+
+  insightId = '6a333202-b1e0-451e-8664-26a1f93c2faf';
+
+  get isSearch() {
+    return this.useCase === 'search';
+  }
+
+  get isInsight() {
+    return this.useCase === 'insight';
+  }
+}
```

Comment:
For future's sake, I think you should use 1 render method here and render different templates instead:

https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.create_render

```suggestion
import {api, LightningElement} from 'lwc';
import {searchTemplate} from './searchTemplate.html';
import {insightTemplate} from './insightTemplate.html';

export default class ExampleUseCase extends LightningElement {
  @api engineId = '';
  @api useCase = '';

  insightId = '6a333202-b1e0-451e-8664-26a1f93c2faf';

  render() {
    if(this.useCase === 'search') {
      return searchTemplate;
    } else if (this.useCase === 'insight') {
      return insightTemplate;
    }
  }
}
```